orchagent/pfcwdorch.cpp

```diff
@@ -312,51 +344,50 @@ PfcWdSwOrch<DropHandler, ForwardHandler>::PfcWdSwOrch(
         DBConnector *db,
         vector<string> &tableNames,
         vector<sai_port_stat_t> portStatIds,
-        vector<sai_queue_stat_t> queueStatIds):
+        vector<sai_queue_stat_t> queueStatIds,
+        vector<sai_queue_attr_t> queueAttrIds):
     PfcWdOrch<DropHandler,
     ForwardHandler>(db, tableNames),
     m_pfcWdDb(new DBConnector(PFC_WD_DB, DBConnector::DEFAULT_UNIXSOCKET, 0)),
     m_pfcWdTable(new ProducerStateTable(m_pfcWdDb.get(), PFC_WD_STATE_TABLE)),
     c_portStatIds(portStatIds),
-    c_queueStatIds(queueStatIds)
+    c_queueStatIds(queueStatIds),
+    c_queueAttrIds(queueAttrIds)
 {
     SWSS_LOG_ENTER();
 
     string platform = getenv("platform") ? getenv("platform") : "";
 
-    if (platform == "")
+    if (platform == MLNX_PLATFORM_SUBSTRING)
     {
-        SWSS_LOG_ERROR("Platform environment variable is not defined");
-        return;
+        SWSS_LOG_NOTICE("Current platform is mlnx");
+    }
+    else
```

Comment:
Why else? We must not default to any platform.
And I don't really understand why we need this kind of notice message.

Reply:
Removed notice message.